### PR TITLE
Array items events for table editor + test + documentation

### PR DESCRIPTION
**Pull request by germanbisurgi**

### CODE REVIEW

File: src/editors/array.js [hunk 1]
```diff
@@ -595,13 +596,7 @@ JSONEditor.defaults.editors.array = JSONEditor.AbstractEditor.extend({
...
         self.onChange(true);
 
-        self.jsoneditor.trigger('move');
-
-        // TODO we can remove this eventually
-        self.rows.forEach(function (row) {
-          row.onMove();
-        });
-
+        self.jsoneditor.trigger('moveRow');
```
Comment (schmunk42) [inline src/editors/array.js]: Do we still have `move` events? If so, where?

**Comment by pmk65** [comment]
I get an error when I try checking this out:
```
CONFLICT (content): Merge conflict in src/editors/array.js
Patch failed at 0001 json editor move event where child editors can subscribe
```

File: src/editors/array.js (hunk 1 above)
Comment (germanbisurgi) [reply src/editors/array.js]: the `row.onMove();`  was the first attempt and was never used in production nor documented. 

s`elf.jsoneditor.trigger('moveRow');` ist already implemented and tested in array editor. It was missing when arrays editors have the table format (table editor).

**Comment by pmk65** [comment]
@germanbisurgi Any chance we can get the conflicts resolved and include it in the develop/2.x branch?

**Comment by pmk65** [comment]
What's the status on this?

**Comment by germanbisurgi** [comment]
@pmk65 for what i see:
- the array events are already merged
- the same events are missing in the table editor (array editor clone with table abilities)
- i did not have merge conflicts when i shipped the pull request. Could you please show me where they are?
- i do not see the "tests/pages/table-move-events.html". Are we working on different branches?

I can send another pull request for tables events if needed.

**Comment by germanbisurgi** [comment]
bump

**Comment by germanbisurgi** [comment]
I submitted a new PR #541 that do not cause any merge conflicts with master or dev/2x

**Comment by germanbisurgi** [comment]
closing this because this fix/feature was merged in another PR

**Pull request closed without merging**